app/views/pages/_v1_editor_guide_preamble.html.erb

```diff
@@ -19,9 +19,9 @@
   <li>When you're ready to publish, set the published variable to <strong>true.</strong></li>
 </ul>
 
-<p><em><b>We have two editor versions</b>. If you'd prefer to edit title and tags etc. as separate fields, switch to the "rich + markdown" option in <a href="/settings/customization">/settings/customization</a>. Otherwise, continue:</em></p>
+<p><em><b>We have two editor versions</b>. If you'd prefer to edit title and tags etc. as separate fields, switch to the "rich + markdown" option in <a href="<%= user_settings_path(:customization) %>">Settings → Customization</a>. Otherwise, continue:</em></p>
```

Comment:
Should we be using `&#8594;` rather than the `→` here? 👀

Reply:
I changed my mind on that a few times over the years, recently rekindled by https://github.com/hail2u/html-best-practices#dont-use-character-references-as-much-as-possible. `→` is UTF8 encodable (which is how we declare the HTML page to be), and since we do not target old browsers I think we're safe 🙌